test/integration/resize_test.go

```diff
@@ -28,6 +29,21 @@ var _ = Describe("vary VM parameters: memory cpus, disk", Label("openshift-prese
 			}
 		})
 
+		It("login to cluster using crc-admin context", func() {
+			cmd := exec.Command("oc", "login", "use-context", "crc-admin")
```

Comment:
without the eval does it able to detect the `oc` binary? As per `exec.Command` doc 
```
Command uses LookPath to resolve name to a complete path if possible. Otherwise it uses name directly as Path.
```
which means if `oc` is not present in the os.Path then this might even break which might be happening as part of CI failure.

Reply:
Sorry, that was me not checking after myself! Another mistakes was using `oc login` instead of `oc config`. Should be corrected now. I checked locally after removing `oc` from PATH.